code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -342,6 +350,10 @@
 	var/fortify = 0
 	var/crest_defense = 0
 
+	// Gorger vars
+	///Cooldown for using rejuvenate on self
+	COOLDOWN_DECLARE(rejuvenate_self_cooldown)
```

Comment:
make this a timer cooldown?

Reply:
How would it benefit from being a timer cooldown?

Reply:
changed to timer